src/libs/ReportUtils.ts

```diff
@@ -1728,7 +1725,7 @@ function getMoneyRequestReportName(report: OnyxEntry<Report>, policy: OnyxEntry<
         return Localize.translateLocal('iou.payerSpentAmount', {payer: payerName, amount: formattedAmount});
     }
 
-    if (!!report?.hasOutstandingIOU || isDraftExpenseReport(report) || moneyRequestTotal === 0) {
+    if ((report?.stateNum === CONST.REPORT.STATE_NUM.PROCESSING && report?.statusNum === CONST.REPORT.STATUS.SUBMITTED) || isDraftExpenseReport(report) || moneyRequestTotal === 0) {
```

Comment:
I understand that this is the essential line, right?

Reply:
Good that you updated the tests 👍

Reply:
Yeah, @rezkiy37 could you add ReportUtils method which will be called: `isProcessingReport` which will do this state/ status check? thanks

Reply:
Yes, sure.

Reply:
Done - https://github.com/Expensify/App/pull/32653/commits/9939213804d745fb5950f69179736a645eabb6be.